swift/apple/FirezoneNetworkExtension/SystemConfigurationResolvers.swift

```diff
@@ -0,0 +1,124 @@
+//
+//  SystemConfiguration.swift
+//  FirezoneNetworkExtensionmacOS
+//
+//  Created by Jamil Bou Kheir on 2/26/24.
+//
+
+import FirezoneKit
+import Foundation
+import SystemConfiguration
+
+class SystemConfigurationResolvers {
+  private let logger: AppLogger
+  private var dynamicStore: SCDynamicStore?
+
+  // Arbitrary name for the connection to the store
+  private let storeName = "dev.firezone.firezone.dns" as CFString
```

Comment:
If we have to provide a name, does that means the system doesn't check our bundle ID or process ID for sandboxing? Could another app write things into our dynamic store?